sample-lnd.conf

```diff
@@ -1131,6 +1131,9 @@ litecoin.node=ltcd
 ; If non zero, LND will use this as peer port for the embedded etcd instance.
 ; db.etcd.embedded_peer_port=1235
 
+; If set the embedded etcd instance will log to the specified file.
+; db.etcd.embedded_log_file=/path/etcd.log
```

Comment:
nit: note that this is only for testing?

Reply:
done